src/SongManager.cpp

```diff
@@ -382,7 +393,7 @@ void SongManager::LoadSongDir( RString sDir, LoadingWindow *ld, bool onlyAdditio
 	StripCvsAndSvn( arrayGroupDirs );
 	StripMacResourceForks( arrayGroupDirs );
 
-	std::map<RString, std::vector<RString>> mapGroupSongDirs;
+	std::unordered_map<RString, std::vector<RString>> mapGroupSongDirs;

```

Comment:
No change here, but more of TMYK, the map was intentional originally so that songs are loaded in alphabetical order which is nice for the loading window, but since we're moving to threading that's no longer guaranteed.